### DISCUSSION TITLE
Feature request: Joomla 4.0 plugin

### DISCUSSION THREAD

**Discussion by Ratingthomas** [mosparo/mosparo#98] [Integrations]
Hi,
I would like to use mosparo on Joomla 4.0. Are there any plans for a Joomla 4.0 plugin?

**Comment by zepich** [comment]
Hi @Ratingthomas 

Thank you very much for your post.

In general, we do not have any plans for additional integrations. But going back to the start of this year, we didn't plan to develop an integration for Django, Drupal, or Mautic, so it's a very dynamic process. Before we create a new integration, we decide on the following criteria:

1. Is it possible to add an integration into a system? Some systems do not allow such an adjustment, and therefore we have no option to do that.
2. Is someone interested in this integration? We do not build integrations to have a lot of integrations, but then nobody plans to use it, and it's just 'dead code'.
3. Do we have someone who could help with testing the integration and gives feedback to us? We only sometimes work with, for example, Joomla, so it's always a bit hard to do the right thing and test it correctly.

In your case, I guess criteria two and probably three are fulfilled since you would be interested in the integration and maybe in giving feedback for it.

So we have to check if the integration is possible. Since Joomla is a large system with many extensions, I guess it should be possible.

The other question is which extensions a mosparo integration must be compatible with. For example, we support six different form plugins and WordPress core forms in WordPress. But what about Joomla? Is there a core functionality for forms, and (or) which form extension should be supported?
